[2025-08-03T10:31:02.123+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:30:00+00:00/task_id=fetch_training_data permission to 509
[2025-08-03T10:31:02.124+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:30:00+00:00/task_id=fetch_training_data permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:30:00+00:00/task_id=fetch_training_data'
[2025-08-03T10:31:02.172+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:30:00+00:00/task_id=fetch_training_data permission to 509
[2025-08-03T10:31:02.175+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:30:00+00:00/task_id=fetch_training_data permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:30:00+00:00/task_id=fetch_training_data'
[2025-08-03T10:31:02.239+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: train_predict_to_db.fetch_training_data scheduled__2025-08-03T10:30:00+00:00 [queued]>
[2025-08-03T10:31:02.249+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: train_predict_to_db.fetch_training_data scheduled__2025-08-03T10:30:00+00:00 [queued]>
[2025-08-03T10:31:02.250+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-08-03T10:31:02.273+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): fetch_training_data> on 2025-08-03 10:30:00+00:00
[2025-08-03T10:31:02.284+0000] {standard_task_runner.py:57} INFO - Started process 339 to run task
[2025-08-03T10:31:02.289+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'train_predict_to_db', 'fetch_training_data', 'scheduled__2025-08-03T10:30:00+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/train_predict_to_db_dag.py', '--cfg-path', '/tmp/tmpnxu7nq_y']
[2025-08-03T10:31:02.293+0000] {standard_task_runner.py:85} INFO - Job 20: Subtask fetch_training_data
[2025-08-03T10:31:02.366+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:30:00+00:00/task_id=fetch_training_data permission to 509
[2025-08-03T10:31:02.367+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:30:00+00:00/task_id=fetch_training_data permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-08-03T10:30:00+00:00/task_id=fetch_training_data'
[2025-08-03T10:31:02.371+0000] {task_command.py:416} INFO - Running <TaskInstance: train_predict_to_db.fetch_training_data scheduled__2025-08-03T10:30:00+00:00 [running]> on host 407839faddcf
[2025-08-03T10:31:02.532+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='train_predict_to_db' AIRFLOW_CTX_TASK_ID='fetch_training_data' AIRFLOW_CTX_EXECUTION_DATE='2025-08-03T10:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-08-03T10:30:00+00:00'
[2025-08-03T10:31:02.546+0000] {db_utils.py:17} INFO - Connected to PostgreSQL
[2025-08-03T10:31:03.763+0000] {train_predict_to_db_dag.py:29} INFO - Fetched training data tail:
            id  store  ...  productname                 created_at
100515  155955    431  ...    product_A 2025-08-03 10:30:13.345847
100516  155956    432  ...    product_A 2025-08-03 10:30:23.365691
100517  155957    426  ...    product_A 2025-08-03 10:30:33.098378
100518  155958    486  ...    product_A 2025-08-03 10:30:43.114728
100519  155959    451  ...    product_A 2025-08-03 10:30:53.117113

[5 rows x 12 columns]
[2025-08-03T10:31:03.810+0000] {python.py:194} INFO - Done. Returned value was: {'store': 370, 'start_date': '2025-05-05', 'end_date': '2025-08-03'}
[2025-08-03T10:31:03.854+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=train_predict_to_db, task_id=fetch_training_data, execution_date=20250803T103000, start_date=20250803T103102, end_date=20250803T103103
[2025-08-03T10:31:03.925+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-08-03T10:31:03.959+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
